### PR TITLE
CSM-1032: Escape dbSchemaName

### PR DESCRIPTION
**Pull request by sgarg-CS**

### CODE REVIEW

File: database-commons/src/main/java/io/cdap/plugin/db/sink/AbstractDBSink.java [hunk 1]
```diff
@@ -467,12 +467,14 @@ public abstract static class DBSinkConfig extends DBConfig implements DatabaseSi
     @Description("Name of the database schema of table.")
     @Macro
     @Nullable
-    private String dbSchemaName;
+    public String dbSchemaName;
```
Comment (prince-cs) [inline database-commons/src/main/java/io/cdap/plugin/db/sink/AbstractDBSink.java:470]: why did we change the scope of this variable?

Comment (sgarg-CS) [reply database-commons/src/main/java/io/cdap/plugin/db/sink/AbstractDBSink.java:470]: The variable `dbSchemaName` is directly accessed at two places outside the package.

1. `AuroraPostgresSinkConfig.getEscapedDbSchemaName()`

```
    public String getEscapedDbSchemaName() {
      return ESCAPE_CHAR + dbSchemaName + ESCAPE_CHAR;
    } 
```

2. `SapHanaSinkConfig.getEscapedDbSchemaName()`

```
    public String getEscapedDbSchemaName() {
      return ESCAPE_CHAR + dbSchemaName + ESCAPE_CHAR;
    }
```

Comment (prince-cs) [reply database-commons/src/main/java/io/cdap/plugin/db/sink/AbstractDBSink.java:470]: okay..

**Review by prince-cs — APPROVED** [review]
LGTM

**Pull request merged**